src/server/routes/api/document/index.js

```diff
@@ -8,8 +8,8 @@ import cytosnap from 'cytosnap';
 import Twitter from 'twitter';
 import LRUCache from 'lru-cache';
 import emailRegex from 'email-regex';
-// import url from 'url';
 import fs from 'fs';
+import { URLSearchParams } from  'url';
 
 import { exportToZip, EXPORT_TYPES } from './export';
```

Comment:
This file needs the conflicts to be resolved

Reply:
@maxkfranz I resolved the conflict and merged the current unstable in this branch.

Reply:
Great, thanks Metin